schemas/Organization.json

```
@@ -29,6 +29,48 @@
           "example": "https://mycallbackurlhandler.com/canopy"
         }
       }
+    },
+    "config": {
+      "description": "Configurations for integrations for your organization.",
+      "type": "object",
+      "properties": {
+        "issuer_processor": {
+          "type": "object",
+          "properties": {
+            "name": {
+              "type": "string",
+              "example": "PRIVACY",
+              "description": "The name of the active configured issuer-processor for your organization."
+            },
+            "valid_api_key": {
+              "type": "boolean",
+              "example": true,
+              "description": "Indicates whether there is a valid API key configured for your issuer-processor."
```

```suggestion
              "description": "Indicates whether the API key configured for your issuer-processor is valid."
```

actually...hmm. do we actually verify api key validity? or is this just checking whether an api key exists.

---

cc @kpamaran

---

existence check atm, there's a plan for the MW to do the easiest get call for each api key